src/neural/cuda/network_cudnn.cc

```diff
@@ -173,6 +173,9 @@ class CudnnNetwork : public Network {
     // 0. Process weights.
     processConvBlock(weights.input, true);
     for (auto i = size_t{0}; i < numBlocks_; i++) {
+      if (weights.residual[i].has_se) {
```

Comment:
It should not be checked at this level. There is `network_format()` check, and LegacyWeights should not be changed anyway.